### DISCUSSION TITLE
What is the best way to build an Backend in Deno?

### DISCUSSION THREAD

**Discussion by FleetAdmiralJakob** [denoland/deno#20177] [General]
Like Nest.js in Node world

**Comment by markthree** [comment]
> Like Nest.js in Node world

Maybe you want to find [danet](https://github.com/Savory/Danet)

**Comment by jollytoad** [comment]
Start with [`Deno.serve`](https://deno.land/manual/runtime/http_server_apis) and work up from there, solve your problems and needs as you encounter them.
